{
    "http://ivs.ipea.gov.br/ivs_componentes/grid/ (IDHM)": "Traceback (most recent call last):\n  File \"c:\\Users\\Daniel\\Documents\\GitHub\\T25\\Scripts\\Daniel\\TESTS.py\", line 31, in <module>\n    driver.wait('/html/body/div[3]/div[2]/div[1]/fieldset/div/div/p[1]/div[1]/span/div/button')\n  File \"c:\\Users\\Daniel\\Documents\\GitHub\\T25\\Scripts\\Daniel\\functions.py\", line 241, in wait\n    WebDriverWait(self.browser, 600).until(\n  File \"C:\\Users\\Daniel\\Documents\\GitHub\\T25\\venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\nselenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00007FF73340CC05+28741]\n\t(No symbol) [0x00007FF73336FED0]\n\t(No symbol) [0x00007FF733204FBA]\n\t(No symbol) [0x00007FF733259256]\n\t(No symbol) [0x00007FF73325948C]\n\t(No symbol) [0x00007FF7332A3267]\n\t(No symbol) [0x00007FF73327FE6F]\n\t(No symbol) [0x00007FF73329FFE1]\n\t(No symbol) [0x00007FF73327FBD3]\n\t(No symbol) [0x00007FF733249F43]\n\t(No symbol) [0x00007FF73324B2D1]\n\tGetHandleVerifier [0x00007FF73373DE3D+3375741]\n\tGetHandleVerifier [0x00007FF73375096A+3452330]\n\tGetHandleVerifier [0x00007FF73374671D+3410781]\n\tGetHandleVerifier [0x00007FF7334D653B+854395]\n\t(No symbol) [0x00007FF73337BE0F]\n\t(No symbol) [0x00007FF733377714]\n\t(No symbol) [0x00007FF7333778AD]\n\t(No symbol) [0x00007FF733366189]\n\tBaseThreadInitThunk [0x00007FFDD483259D+29]\n\tRtlUserThreadStart [0x00007FFDD504AF38+40]\n\n",
    "Gr√°fico 20.2": "Traceback (most recent call last):\n  File \"c:\\Users\\Daniel\\Documents\\GitHub\\T25\\Scripts\\Daniel\\TESTS.py\", line 120, in <module>\n    data = c.open_file(dbs_path, 'ipea (IDHM).csv', 'csv')\n  File \"c:\\Users\\Daniel\\Documents\\GitHub\\T25\\Scripts\\Daniel\\functions.py\", line 87, in open_file\n    dataframe = pd.read_csv(os.path.join(dir_path, file_path),\n  File \"C:\\Users\\Daniel\\Documents\\GitHub\\T25\\venv\\lib\\site-packages\\pandas\\io\\parsers\\readers.py\", line 1026, in read_csv\n    return _read(filepath_or_buffer, kwds)\n  File \"C:\\Users\\Daniel\\Documents\\GitHub\\T25\\venv\\lib\\site-packages\\pandas\\io\\parsers\\readers.py\", line 620, in _read\n    parser = TextFileReader(filepath_or_buffer, **kwds)\n  File \"C:\\Users\\Daniel\\Documents\\GitHub\\T25\\venv\\lib\\site-packages\\pandas\\io\\parsers\\readers.py\", line 1620, in __init__\n    self._engine = self._make_engine(f, self.engine)\n  File \"C:\\Users\\Daniel\\Documents\\GitHub\\T25\\venv\\lib\\site-packages\\pandas\\io\\parsers\\readers.py\", line 1880, in _make_engine\n    self.handles = get_handle(\n  File \"C:\\Users\\Daniel\\Documents\\GitHub\\T25\\venv\\lib\\site-packages\\pandas\\io\\common.py\", line 882, in get_handle\n    handle = open(handle, ioargs.mode)\nFileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\Daniel\\\\AppData\\\\Local\\\\Temp\\\\tmpeqbrr7_9\\\\ipea (IDHM).csv'\n"
}